3rdparty/find_dependencies.cmake

```diff
@@ -491,7 +491,7 @@ if(NOT USE_SYSTEM_JPEG)
     import_3rdparty_library(3rdparty_jpeg
         INCLUDE_DIRS ${CMAKE_CURRENT_BINARY_DIR}/libjpeg-turbo-install/include/
         LIBRARIES ${JPEG_TURBO_LIBRARIES}
-        LIB_DIR ${CMAKE_CURRENT_BINARY_DIR}/libjpeg-turbo-install/lib
+        LIB_DIR ${CMAKE_CURRENT_BINARY_DIR}/libjpeg-turbo-install/${CMAKE_INSTALL_LIBDIR}
```

Comment:
This is a good idea. Change it to `Open3D_INSTALL_LIB_DIR` to make it work for Windows as well

See:
- https://github.com/intel-isl/Open3D/blob/103caa859fe0a6c928b898213da4a88517595135/CMakeLists.txt#L232
- https://github.com/intel-isl/Open3D/blob/103caa859fe0a6c928b898213da4a88517595135/CMakeLists.txt#L237

Reply:
Thank you for your advice.

Changed from `CMAKE_INSTALL_LIBDIR` to `Open3D_INSTALL_LIB_DIR`.
I also replaced `3rdparty/librealsense/librealsense.cmake` with `Open3D_INSTALL_LIB_DIR`.